README.md

```diff
@@ -208,6 +208,12 @@ Allows custom monitoring events to be added into the Node Application Metrics ag
 * `type` (String) the name you wish to use for the data. A subsequent event of that type will be raised, allowing callbacks to be registered for it.
 * `data` (Object) the data to be made available with the event. The object must not contain circular references, and by convention should contain a `time` value representing the milliseconds when the event occurred.
 
+### appmetrics.writeHeapSnapshot([filename],[callback])
+Dumps the v8 heap via `heapdump`. For more information, see [the heapdump README](https://github.com/bnoordhuis/node-heapdump/blob/master/README.md)
+
+### appmetrics.triggerNodeReport()
```

Comment:
@sam-github yep, I agree, especially as there is going to be additional api's on nodereport that we will want to call so that we can inject additional information